cg/cli/upload/vogue.py

```diff
@@ -105,6 +108,8 @@ def samples(context, days: int):
 def bioinfo(context, case_name, cleanup, target_load, dry):
     """Load bioinfo case results to the trending database"""
 
+    valid_workflows = ["mip"]
+
```

Comment:
Related to what I suggested below. To make code more transparent I suggest to pick out what is needed from the context and send only that to the helper functions.

```python
hk_api = context.obj['db']
load_bioinfo_raw_inputs["samples"] = _get_samples(hk_api, case_name)
```
etc..

Reply:
I updated the code according to your suggestion. Let me know if this is A-ok.